source/mainnet/smart-contracts/guides/contract-dev-guides.rst

```diff
@@ -6,6 +6,8 @@ Contract development guides
 
 The contract development guides help you get started writing smart contracts. Here you have guides to help you set up your tools, test your contract, and more.
 
+Before starting, it is a good idea to read the :ref:`Smart contracts best practices<sc-best-practices>`.
```

Comment:
I think this is not very good advice. If we assume that these are guides for people who are not familiar with smart contracts, then reading this best practices first is not going to help them a lot. A lot of them are not going to be understandable before they have a reasonable idea of how smart contracts look.